.github/workflows/components-contrib.yml

```diff
@@ -21,11 +21,11 @@ jobs:
     name: Build ${{ matrix.target_os }}_${{ matrix.target_arch }} binaries
     runs-on: ${{ matrix.os }}
     env:
-      GOVER: 1.13.7
+      GOVER: 1.14
       GOOS: ${{ matrix.target_os }}
       GOARCH: ${{ matrix.target_arch }}
       GOPROXY: https://proxy.golang.org
-      GOLANGCI_LINT_VER: v1.23.2
+      GOLANGCI_LINT_VER: v1.26.0
```

Comment:
Can we upgrade to 1.27.0 ? https://github.com/golangci/golangci-lint/releases/tag/v1.27.0

Reply:
oh. I saw your comment.